src/linter.rs

```diff
@@ -415,10 +416,20 @@ impl Linter {
       plugin_codes: HashSet::new(),
     };
 
-    // Run builtin rules
-    for rule in &self.rules {
-      rule.lint_program(&mut context, program);
-    }
+    let program_info = ProgramInfo {
+      program,
+      source_file: None,
+      tokens: None,
+      // TODO(@magurotuna): let it have SingleThreadedComments
+      comments: None,
```

Comment:
This should be fixed before landing?

Reply:
Not really, I think it can be fixed when we try to refactor some rules that look at comments to use dprint-swc-ecma-ast-view, so it's okay for now.